cf-deployment.yml

```diff
@@ -870,8 +870,10 @@ instance_groups:
         - name: blobstore
           port: 8080
```

Comment:
You can remove this now ✨

Reply:
During a past iteration of this effort, there [was concern](https://www.pivotaltracker.com/story/show/164937832/comments/202023498) about removing the non-TLS ports, since they could be cached in some ancient LRPs. 

We can re-evaluate the risk here, since time has continued to progress. cc @tcdowney

Reply:
I think gorouter will just disregard the `port` param if you provide `tls_port`.
* gorouter makes the endpoint [here](https://github.com/cloudfoundry/gorouter/blob/43d4f29110f9f007b4ce59939ea9add3b537df92/mbus/subscriber.go#L47).
* and it calculated port [here](https://github.com/cloudfoundry/gorouter/blob/43d4f29110f9f007b4ce59939ea9add3b537df92/mbus/subscriber.go#L85-L90) and only returns one or the other.


If you want both endpoints to still exist, you will need to add it as a separate route in the array here.

Reply:
Oh, I see. I thought you were talking about removing the actual port, not just the register configuration.